.phpmetrics.yml

```
@@ -0,0 +1,94 @@
+excludes:
+    - ./tests
+    - ./vendor
+    - ./examples
+    - ./tools
+    - ./docs
+
+report:
+    html: ./build/metrics/
+    json: ./build/metrics.json
+    csv: ./build/metrics.csv
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)